jnigen/lib/src/tools/build_summarizer.dart

```diff
@@ -2,6 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+// coverage:ignore-file
```

Comment:
Ignoring this file until #43 is addressed. Majority of this file will be never run under test so..

Reply:
Please add a TODO comment here.